File: shared/src/main/java/com/jawnnypoo/openmeh/shared/api/MehResponse.java
package com.jawnnypoo.openmeh.shared.api;

import com.jawnnypoo.openmeh.shared.model.Deal;
import com.jawnnypoo.openmeh.shared.model.Video;

/**
 * The actual response we get when pulling in the meh deal
 */
public class MehResponse {
    Deal deal;
    Video video;

    public MehResponse() {}

    public Deal getDeal() {
        return deal;
    }

    public Video getVideo() {
        return video;
    }
}

File: shared/src/main/java/com/jawnnypoo/openmeh/shared/response/MehResponse.kt
package com.jawnnypoo.openmeh.shared.response

import com.jawnnypoo.openmeh.shared.model.Deal
import com.jawnnypoo.openmeh.shared.model.Video
import com.squareup.moshi.Json

/**
 * The actual response we get when pulling in the meh deal
 */
class MehResponse {
    @Json(name = "deal") var deal: Deal? = null
    @Json(name = "video") var video: Video? = null
}
